File: guanabara-java/src/com/danillo/algoritmos/lista06/Exercicio62.java
package com.danillo.algoritmos.lista06;

import java.util.Scanner;

public class Exercicio62 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        int quantidadeIdadesDigitadas = 0, soma = 0, pessoasCom21AnosOuMais = 0;
        char continuar;

        do{
            System.out.println("Insira a idade: ");
            int idade = entrada.nextInt();

            if(idade >= 21){
                pessoasCom21AnosOuMais++;
            }

            soma = soma + idade;
            quantidadeIdadesDigitadas++;

            System.out.println("Deseja continuar? (S ou N)");
            continuar = entrada.next().charAt(0);
        }while(continuar == 'S');

        float media = (float) soma/quantidadeIdadesDigitadas;

        System.out.println("Quantidade de idades digitadas: " +quantidadeIdadesDigitadas+
                           "\nMédia entre as idades digitadas: " +media+
                           "\nPessoas com 21 anos ou mais: " +pessoasCom21AnosOuMais);
    }
}

File: guanabara-python/src/lista06/exercicio62.py
i = 0
soma_idade = 0
pessoas_mais_21 = 0

while True:
    idade = int(input("Insira a idade: "))

    soma_idade = soma_idade + idade

    if idade >= 21:
        pessoas_mais_21 = pessoas_mais_21 + 1

    i = i + 1

    continuar = int(input("Deseja continuar: 1 - Sim | 2 - Não "))
    if continuar == 2:
        break

media_idade = soma_idade / i

print(f"Idades digitadas: {i}.\nMédias das idades: {media_idade}.\nPessoas com 21 anos ou mais: {pessoas_mais_21}.")
